views/albums.hbs

```diff
@@ -0,0 +1,11 @@
+<nav class="albums-for"><h1>Albums</h1></nav>
```

Comment:
Answering your question, you needed to use `dott notation` and probably some `console.logs` on your `root get method` to find the correct path to the `name`. 
Remember that we don't have access to `bracket notation` in handlebars, so, we use the `dott notation` to enter the info inside arrays.
`<nav class='albums-for'><h1>Albums for: {{result.items.0.artists.0.name}}</h1></nav>`